mlrun/projects/project.py

```diff
@@ -2368,26 +2343,6 @@ def save_workflow(self, name, target, artifact_path=None, ttl=None):
         workflow_engine = get_workflow_engine(workflow_spec.engine)
         workflow_engine.save(self, workflow_spec, target, artifact_path=artifact_path)
 
-    def get_run_status(
-        self,
-        run,
-        timeout=None,
-        expected_statuses=None,
-        notifiers: CustomNotificationPusher = None,
-    ):
-        warnings.warn(
-            "This is deprecated in 1.3.0, and will be removed in 1.5.0. "
```

Comment:
do not deprecate that still, move it to 1.6.0
get run status returns state, errors, text and stuff which `project.run` doesnot.
this is a convenient method to return an existing run status and does not have a real replacement